src/StatusQ/Layout/StatusAppLayout.qml

```diff
@@ -1,49 +1,88 @@
 import QtQuick 2.13
-import QtQuick.Layouts 1.13
+import QtQuick.Layouts 1.14
 import QtQuick.Controls 2.13
 
-import StatusQ.Core 0.1
+import StatusQ.Components 0.1
 import StatusQ.Core.Theme 0.1
 
-Rectangle {
-    id: statusAppLayout
-
-    implicitWidth: 900
+SplitView {
+    id: root
+    implicitWidth: 822
     implicitHeight: 600
 
-    color: Theme.palette.statusAppLayout.backgroundColor
-    
-    property StatusAppNavBar appNavBar
-    property Item appView
+    handle: Item { }
+
+    property Item headerContent
+    property Item leftPanel
+    property Item centerPanel
+    property Component rightPanel
```

Comment:
I find it kinda confusing `leftPanel` is an `Item` and `rightPanel` is a `Component`

Reply:
that was back when StatusThreePanelLayout was created there was the comment to make right panel a component to be loaded on demand, since we do not have it in all cases (is used only in members list as of now).

Reply:
Ok